rust/cache/src/bin/cops-memory-cache-config-writer.rs

```diff
@@ -0,0 +1,10 @@
+use clap::Parser;
+
+use chroma_cache::{CacheConfig, FoyerCacheConfig};
+
+fn main() {
```

Comment:
nit / thought: might be nicer to have one binary that takes a flag for which config it generates

Reply:
I chose to make it this way as it's clear from the tool which output you'll get.  This allows us to have one tool per enum variant with exactly the options said variant needs.  I'm not wedded to this idea, though.

Reply:
good with me